src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -393,92 +393,118 @@ public static class PublishedContentExtensions
         }
 
         /// <summary>
-        /// Returns the resolved value for the given type and property.
+        /// Returns the processed value for the given type and property.
         /// </summary>
-        /// <param name="content">The <see cref="IPublishedContent"/> to convert.</param>
-        /// <param name="culture">The <see cref="CultureInfo"/></param>
-        /// <param name="propertyInfo">The <see cref="PropertyInfo"/> property info associated with the type.</param>
+        /// <param name="content">The <see cref="IPublishedContent" /> to convert.</param>
+        /// <param name="culture">The <see cref="CultureInfo" /></param>
+        /// <param name="type">The type.</param>
+        /// <param name="propertyInfo">The <see cref="PropertyInfo" /> property info associated with the type.</param>
         /// <param name="instance">The instance to assign the value to.</param>
-        /// <param name="valueResolverContexts">
-        /// A collection of <see cref="DittoValueResolverContext"/> entities to use whilst resolving values.
-        /// </param>
-        /// <returns>The <see cref="object"/> representing the Umbraco value.</returns>
-        private static object GetResolvedValue(
+        /// <param name="processorContexts">A collection of <see cref="DittoProcessorContext" /> entities to use whilst processing values.</param>
+        /// <returns>
+        /// The <see cref="object" /> representing the Umbraco value.
+        /// </returns>
+        private static object GetProcessedValue(
             IPublishedContent content,
             CultureInfo culture,
+            Type type,
             PropertyInfo propertyInfo,
             object instance,
-            IEnumerable<DittoValueResolverContext> valueResolverContexts = null)
+            IEnumerable<DittoProcessorContext> processorContexts = null)
         {
             // Check the property for an associated value attribute, otherwise fall-back on expected behaviour.
-            var valueAttr = propertyInfo.GetCustomAttribute<DittoValueResolverAttribute>(true);
+            var valueAttrs = propertyInfo.GetCustomAttributes<DittoProcessorAttribute>(true)
+                .OrderBy(x => x.Order)
+                .ToList();
 
-            if (valueAttr == null)
+            if (!valueAttrs.Any())
```

Comment:
Yea, cos I call ToArray, you always get an array. Well, that's what VS is telling me anyway :)